Missing "Loader=yaml.FullLoader" parameter in generate-binary-license and generate-binary-notice py scripts

### Affected Version

- `0.22.0`

### Description

- These error messages appear when building Druid with the command `mvn clean install -X -Pdist -DskipTests` using Maven `3.8.3` :
```
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (generate-binary-license) on project distribution: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (generate-binary-license) on project distribution: Command execution failed.

[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (generate-binary-notice) on project distribution: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (generate-binary-notice) on project distribution: Command execution failed.
```
### Debugging

There is an issue in both of these files:
- https://github.com/apache/druid/blob/druid-0.22.0/distribution/bin/generate-binary-license.py
- https://github.com/apache/druid/blob/druid-0.22.0/distribution/bin/generate-binary-notice.py

`yaml.load_all` is missing `Loader` parameter

--- a/distribution/bin/generate-binary-license.py
+++ b/distribution/bin/generate-binary-license.py
@@ -137,7 +137,7 @@ def generate_license(apache_license_v2, license_yaml):
     # Print Apache license first.
     print_outfile(apache_license_v2)
     with open(license_yaml, encoding='utf-8') as registry_file:
-        licenses_list = list(yaml.load_all(registry_file))
+        licenses_list = list(yaml.load_all(registry_file, Loader=yaml.FullLoader))
 
     # Group licenses by license_name, license_category, and then module.
     licenses_map = {}

--- a/distribution/bin/generate-binary-notice.py
+++ b/distribution/bin/generate-binary-notice.py
@@ -57,7 +57,7 @@ def generate_notice(source_notice, dependences_yaml):
     # Print Apache license first.
     print_outfile(source_notice)
     with open(dependences_yaml, encoding='utf-8') as registry_file:
-        dependencies = list(yaml.load_all(registry_file))
+        dependencies = list(yaml.load_all(registry_file, Loader=yaml.FullLoader))
 
     # Group dependencies by module
     modules_map = defaultdict(list)

